apps/omg_api/lib/state/core.ex

```diff
@@ -308,7 +317,9 @@ defmodule OMG.API.State.Core do
     db_updates_spent_utxos =
       txs
       |> Enum.flat_map(fn %Transaction.Recovered{signed_tx: %Transaction.Signed{raw_tx: tx}} ->
-        [Utxo.position(tx.blknum1, tx.txindex1, tx.oindex1), Utxo.position(tx.blknum2, tx.txindex2, tx.oindex2)]
+        tx
+        |> Transaction.get_inputs()
+        |> Enum.map(fn %{blknum: blknum, txindex: txindex, oindex: oindex} -> Utxo.position(blknum, txindex, oindex) end)
```

Comment:
yeah, `Utxo.Position` should be returned `Transaction` functions, and here in `State.Core` we never use anyting else for utxo position

Reply:
There is one use, when we filter inputs by block number only. Matching by Utxo.position is cumbersome, since you need it to write it with placeholders. I know that Paweł T. also has a use similar use case.

Reply:
can you point me to that?